### PR TITLE
[MODULAR][READY]Soft-Reverts 18747

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

[#18747](https://github.com/Skyrat-SS13/Skyrat-tg/pull/18747) was the beret one. Re-adds the Atmos beret as everyone has a beret here.

Personally, I'd add the badge beret to loadout as well but...Something something scope.

## How This Contributes To The Skyrat Roleplay Experience

Every other job/department (sans Brig Officer, Barber and the rest of Service) have berets.
I also believe this was requested by Golden, correct me on that if you will. Most information I get is second hand.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
 
image

</details>

## Changelog

:cl: Cursor
add: Atmos Berets are back.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm [hunk 1]
```diff
@@ -544,6 +544,11 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
 *	JOB BERETS
 */
 
+/datum/loadout_item/head/atmos_beret
+	name = "Atmospherics Beret"
+	item_path = /obj/item/clothing/head/beret/atmos
+	restricted_roles = list(JOB_STATION_ENGINEER, JOB_ATMOSPHERIC_TECHNICIAN, JOB_CHIEF_ENGINEER, JOB_ENGINEERING_GUARD)
```
Comment (tf-4) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm]:
```suggestion
	restricted_roles = list(
		JOB_ATMOSPHERIC_TECHNICIAN,
		JOB_CHIEF_ENGINEER,
	)
```

**Review by tf-4 — DISMISSED** [review]
I agree with the original PR honestly. I don't see why everyone should have berets.
And "everyone else has one" is not a reason for everyone to have one

**Review by GoldenAlpharex — COMMENTED** [review]
As the person that's added most of these, I still don't see berets as anything "milsim" or whatever, you know how little I actually care about that. They're just one of those very good-looking headwear that goes well with ear sprites, which isn't true of a lot of other headwear.

GAGS isn't entirely all about just letting people customize the items themselves, it's also to provide an easy way to add variants already baked in that have specific colors you want them to have, i.e. here the colors for the atmos beret.

Just do the change tf4 suggested, and it's good on my side.

**Comment by theselfish** [comment]
@GoldenAlpharex Done. I was out getting messages, apologies.

**Comment by theselfish** [comment]
> I agree with the original PR honestly. I don't see why everyone should have berets. And "everyone else has one" is not a reason for everyone to have one

I don't care either way, in truth. I think the simpler solution would be to add a badged beret option to the loadout. But I heard that @GoldenAlpharex had intended to re-add it anyway. And I was a coding option.

But, I think singling out Atmos when nearly every other department/job barring a small pool has one is silly.

They are hats. I do not believe a beret is a military item, it is used by the military yes. But so are a great many things. 
Not to engage in whataboutery or going on about cultures that wear berets.

If it's a 'people in an engineering environment wouldn't wear them' you're correct, but if that's the issue, then you have **a lot** of files to gut and uniform/haircut policies to enforce. I know it isn't that. It's late, I'm rambling.

**Comment by Iamgoofball** [comment]
hi, we're cutting back upstream on berets because they're very military aesthetic along with cutting back on clothing bloat, and we keep adding hundreds of beret subtypes for each job when players should just be going to the clothing vendor and getting a GAGS beret of their preferred type(or on skyrat, loadout)

you gamers can probably just stick a gags beret in the loadout and call it a day and it'd suffice

**Comment by theselfish** [comment]
> hi, we're cutting back upstream on berets because they're very military aesthetic along with cutting back on clothing bloat, and we keep adding hundreds of beret subtypes for each job when players should just be going to the clothing vendor and getting a GAGS beret of their preferred type(or on skyrat, loadout)
> 
> you gamers can probably just stick a gags beret in the loadout and call it a day and it'd suffice

Again. That is what I was going to (and might as an alternate PR option) do.
I was merely stealing work from someone else's plate.

**Review by GoldenAlpharex — APPROVED** [review]
The vast majority of people wearing berets in this game don't do it for "milsim", they do it because they just look good.

I don't see any problem with providing people with templates that have already been colored for a specific job/palette, it just makes it less complicated for them to do it themselves.

**Pull request merged**